README.md

```diff
@@ -1,77 +1,373 @@
+
+### Browser
+
+The preferred browser for Vets Who Code is the Google Chrome, due to its popularity and its robust developer tools. To download it please go [here](https://www.google.com/chrome/) and follow the prompts.
+
+
+### Using GitHub
+ We use GitHub extensively here. Below you’ll find the steps to use it within the terminal or command prompt.
+
+1.	 Install Git: Git is a version control system used by GitHub. You can download and install Git for Windows from the official Git website.
+2.	 Configure Git: After installing Git, open the Git Bash application and configure your name and email address using the following commands:
+```console
+git config --global user.name "Your Name"
+git config --global user.email "youremail@example.com"
+```
+
+
+
+3.	Create a repository: A repository is a place where you can store and manage your code. Click on the “New” button and create a new repository.
+4.	Clone the repository: To start working with the repository, you need to clone it to your local machine. You can do this by clicking on the “Code” button and copying the repository URL. Then, open your terminal or command prompt and type `git clone URL-Goes-Here`.
+5.	Make changes to the repository: Once you have cloned the repository, you can change the code using your favorite code editor.
+6.	Add changes to the staging area: To add your changes to the staging area, type `git add` followed by the file name or directory you want to add. You can use `git add .` to add all the changes.
+7.	Commit changes: Once you have added your changes to the staging area, you can commit them by typing `git commit -m` followed by a brief description of the changes you made.
+8.	Push changes: To upload your changes to the GitHub repository, type `git push` followed by the name of the branch to which you want to push the changes.
+9.	Pull changes: If someone else has changed the repository, you can pull their changes to your local machine by typing `git pull`.
+10.	Create a pull request: If you want to suggest changes to someone else’s repository, you can create a pull request by clicking on the “Pull request” button on their repository page.
+11.	Merge changes: The repository owner can merge your changes into their repository by clicking on the “Merge pull request” button.
+
+<h2 id="html">HTML & CSS</h2>
+### HTML && CSS
+HTML (Hypertext Markup Language) is the standard markup language used to create web pages, while CSS (Cascading Style Sheets) is used to style and layout the content. Together, they form the building blocks of most websites.
```

Comment:
> HTML (Hypertext Markup Language)

to `Hypertext Markup Language (HTML)`

> is the standard markup language

I'd just say `standard language`

> CSS (Cascading Style Sheets)

to `Cascading Style Sheets (CSS)`